webapp/src/translations/languages/en.json

```diff
@@ -979,11 +979,11 @@
     "download": "Download",
     "restoreYourWallet": "Restore your wallet from the following backup?",
     "exitBackupWarning": "Exiting your wallet now will remove your wallet from the app. To access this wallet again later, you will need the wallet backup. Do you want to create a wallet backup first before exiting?",
-    "backupAndExit": "Backup and Exit",
+    "backupAndExit": "Back up and Exit",
```

Comment:
Used as a noun, keep it as "Backup and Exit"

Reply:
I disagree. "Back up" is a verb here and really cannot be used as a noun. I suggest the wording "create backup" instead.

Reply:
Thanks for your feedback. This cannot be "create backup" as the wallet will trigger an "exit" to the "Create / Restore Wallet" screen. You could change it to "Create backup and Exit" but it might break UI. 

Kindly understand that you suggestions are taken in consideration and we appreciate it, however if this above suggestion causes a break in the UI, it'll not a major bug, it'll be left unattended for awhile. Something to note when suggesting translation, the shorter & more straight forward the better. Thank you for your understanding.

Reply:
Yes, I changed it to "Create backup and exit." The UI looks okay for me. (The spacing doesn't look perfect to my eye, but nothing is cut off or so.)